### PR TITLE
Prepare release 2.0.0

### PR DESCRIPTION
**Pull request by traylenator**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+## 2017-06-07 - Release 2.0.0
+
+- Data for CentOS and epel yum repositories added.
+- Yum repositories can now be configured via managed_repos and repos parameters. 
+  By default these are not configured.
+- More precise validation of versionlock strings.
+- New config_options hash parameter allows yum configuration to be specified. See
+  README.md for examples.
+- Main class parameters keepcache, debuglevel, exactarch, obsoletes, gpgcheck and
+  installonly_limit are removed. Use config_options hash instead.
```
Comment (bastelfreak) [inline CHANGELOG.md]: @traylenator can you try the new 'bundle exec rake changelog' to generate the changelog?

**Comment by traylenator** [comment]
Something weird with the changelog generation.

**Comment by traylenator** [comment]
So to fix the changelog issue I specified the user which for the general case will be served by
https://github.com/voxpupuli/modulesync_config/pull/354

**Comment by traylenator** [comment]
This this is ready unless there is away to highlight the backwards-incompatible flag in the changelog.

**Pull request merged**